R/prerun.R

```diff
@@ -163,7 +163,7 @@ build_state_output <- function(queue, output, options) {
 create_result <- function(queue, result) {
   task_id <- ids::random_id()
   rrq:::run_task_cleanup(queue$queue$con,
```

Comment:
Can you please spec out what is required here so that we can implement whatever functionality is missing in rrq directly?

Reply:
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4061/Add-a-way-to-save-a-precomputed-result-directly-in-redis